main/templates/main/submission.html

```diff
@@ -134,19 +135,100 @@ <h5>{{c.entry_name}}</h5>
             <div class="col-md-8 col-map col-wide">
               <div id='map' class="map-visualization">
                 <div id="pdf-loading-box" class="map-popup-box">
-                    <p class="mb-0">Loading PDF...</p>
+                  <p class="mb-0">Loading PDF...</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div class="modal fade" id="thanksModal" tabindex="-1" role="dialog" aria-labelledby="thanksModalLabel" aria-hidden="true">
+          <div class="modal-dialog modal-dialog-centered" role="document" id="thanksModalDialog">
+            <div class="modal-content">
+              <div class="modal-header">
+                <h1 class="modal-title text-center font-weight-light" id="thanksModalLabel">Thank you for your submission!</h1>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                  <span aria-hidden="true">&times;</span>
+                </button>
+              </div>
+              <div class="modal-body">
+                <div class="row my-3">
+                  <div class="col-sm my-3 mx-3">
+                    <div class="card border-0">
+                      <img class="card-img-top img-fluid small-icon m-2 h-25 w-50 mx-auto" src="{% static 'img/send.svg' %}"></img>
+                      <div class="card-body text-center d-flex flex-column align-items-center">
+                        <h5>
+                          Visualize/Export
+                        </h5>
+                        <p>
+                          View your community on our live map, and export in PDF or industry standard GeoJSON formats to share with your representatives or mapmakers.
+                        </p>
+                        <div class="text-center">
+                          <a id="card-add-community-btn" class="btn btn-outline-primary mx-2 my-1" href="/submission/{{ map_id }}" role="button">View Community</a>
```

Comment:
One thing I would change is when clicking "view community" instead of requesting that page again, change the logic to just close the modal? The extra request is kinda unnecessary.